controllers/runner_controller.go

```diff
@@ -821,7 +829,8 @@ func (r *RunnerReconciler) newPod(runner v1alpha1.Runner) (corev1.Pod, error) {
 				},
 			},
 			SecurityContext: &corev1.SecurityContext{
-				Privileged: &privileged,
+				Privileged:     &privileged,
+				SELinuxOptions: seLinuxOptions,
```

Comment:
There's also the `Dockerd in Runner Container` mode that runs dockerd in the first container

https://github.com/actions-runner-controller/actions-runner-controller/blob/cb60c1ec3baed8690d4546dacfdfdc564891e852/controllers/runner_controller.go#L693-L696

Probably you'd need to make similar change on that code path, too?

Reply:
Yes - good point!